docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,100 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+## Diagnostic and Recovery Tools
+
+In case of PMEM device restored to healthy state, the ext4 filesystem
+created on each PMEM devices may need to verified and repaired if needed.
+
+* **Make sure that PMEM is not in use and not mounted while doing check or repair.**
+
+## e2fsck
+
```

Comment:
style: no level-3 ### heading between ## above and #### below?

Reply:
It suppose to be level-3. Thanks for catching it.